### ISSUE TITLE
Package.json exports issue with NodeJS v13.12.0

### ISSUE THREAD

**Issue by moollaza** [storeon/svelte#18]
Trying to use this in a Sapper project where I'm using Node v13.12.0

When running my dev server, the following error message prints:

```
Package subpath './package.json' is not defined by "exports" in /root/.../node_modules/@storeon/svelte/package.json
```

After playing around with the package.json, I managed to make the error go away by simply removing the `exports:` from the `package.json`

Not sure if this is a Node issue, or an issue with the package?

Possibly related to https://medium.com/@nodejs/announcing-core-node-js-support-for-ecmascript-modules-c5d6dc29b663?

**Comment by moollaza** [comment]
Just confirmed this isn't an issue with Node v12

**Comment by ai** [comment]
The issue was fixed in fresh `dual-publish` release. We need to republish it.

**Comment by ai** [comment]
> After playing around with the package.json, I managed to make the error go away by simply removing the exports: from the package.json

By removing `exports` you remove ability to load CommonJS files on old Node.js versions

**Comment by distolma** [comment]
Fixed at 0.6.2
